autogen/main/README.md

```diff
@@ -240,6 +240,7 @@ The [project factory](https://github.com/terraform-google-modules/terraform-goog
 In order to execute this module you must have a Service Account with the
 following project roles:
 - roles/compute.viewer
+- roles/compute.securityAdmin
```

Comment:
Can we note this is only required if firewalls are enabled?

Reply:
Done